client/src/lib/dependency-injection/tests/ApolloGraphqlScaffoldingContainer-test.js

```diff
@@ -184,8 +184,8 @@ describe('Dynamic graphql injection', () => {
       singularName: 'Dino',
       pagination: false,
       params: {
-        // Declare that the "id" param is lowercase
-        id: 'ID!'
+        // Declare that the "id" param is uppercase
+        ID: 'ID!'
```

Comment:
Small question, does it still work with lowercase `id`?

Reply:
Yes - that's now the default in fact. See line 175 where another test is updated to account for that.